packages/bsp/common/etc/modprobe.d/r8723bs.conf

```diff
@@ -1 +1 @@
-options r8723bs rtw_power_mgnt=0 rtw_enusbss=0
```

Comment:
I would test that first.

Reply:
Sure, then just keep it for now. 
I don't have such an USB-Adapter so I can not test it myself.

Reply:
I think Tinkerboard or Pinebook has this chip.

Reply:
OPiPrime too ...